tests/python/frontend/pytorch/test_forward.py

```diff
@@ -0,0 +1,608 @@
+    if torch.cuda.is_available():
+        trace = trace.cuda()
+    else:
+        trace = trace.cpu()
+    with TemporaryDirectory() as tmp:
+        path = os.path.join(tmp, 'model.pth')
+        torch.jit.save(trace, path)
+        mod, params = relay.frontend.from_pytorch(trace, input_shapes)
+
+    compiled_input = {input_name: tvm.nd.array(baseline_input.cpu().numpy())}
+
+    with relay.build_config(opt_level=3):
+        relay_graph, relay_lib, relay_params = relay.build(mod, target=TARGET, params=params)
+        relay_model = graph_runtime.create(relay_graph, relay_lib, CTX)
+        relay_model.set_input(**relay_params)
+        relay_model.set_input(**compiled_input)
+        relay_model.run()
+
+    for i, baseline_output in enumerate(baseline_outputs):
+        output_shape = baseline_output.shape
+        compiled_output = relay_model.get_output(
+            i, tvm.nd.array(np.zeros(output_shape).astype(dtype), CTX)).asnumpy()
+
+        compiled_relay_output = relay_model.get_output(
+            i, tvm.nd.array(np.zeros(output_shape).astype(dtype), CTX)).asnumpy()
+
+        assert_shapes_match(baseline_output, compiled_output)
+        tvm.testing.assert_allclose(baseline_output, compiled_output,
+                                    rtol=1e-3, atol=1e-3)
+
+        assert_shapes_match(baseline_output, compiled_relay_output)
+        tvm.testing.assert_allclose(baseline_output, compiled_relay_output,
+                                    rtol=1e-3, atol=1e-3)
+
+    if(test_repeats > 0):
+        thresh = 1e-2
+        units = 1e3
+        thresh = int(thresh * units)
+        input_shapes = list(input_shapes.values())
+
+        compiled_latencies = []
+        baseline_latencies = []
+        speedups = []
+
+        for i in range(0, test_repeats):
+            print("computing compiled latency")
+            compiled_latency = measure_latency(relay_model, input_shapes,
+                                               output_shapes, thresh) * units
+            print(f'Compiled latency is {compiled_latency:.3f} +/- {thresh:d} ms.')
+            print("computing baseline latency")
+            baseline_latency = measure_latency(baseline_model, input_shapes,
+                                               output_shapes, thresh) * units
+
+            print(f'Baseline latency is {baseline_latency:.3f} +/- {thresh:d} ms.')
+
+            speedup = baseline_latency/compiled_latency
+            print(f'Relative speedup is {speedup:.3f}')
+
+            compiled_latencies.append(compiled_latency)
+            baseline_latencies.append(baseline_latency)
+            speedups.append(speedup)
+
+        baseline_latencies_map[model_name] = baseline_latencies
+        compiled_latencies_map[model_name] = compiled_latencies
+        speedups_map[model_name] = speedups
+        model_names.append(model_name)
+
+        print_results()
+
+    from subprocess import call
+    call('rm -rf ~/.torch/models/*', shell=True)
```

Comment:
Unfortunately, there's not a clean way to change this. The ~/.torch/models directory is used by default when using the torchvision classes (https://github.com/pytorch/vision/blob/master/torchvision/models/utils.py#L4, parameter model_dir is not passed in which sets where to save the weights). A workaround would be to update the testscript to manually load the parameters in our code rather than just using torchvision but this could potentially introduce a lot of copied code from torchvision as some models do some extra steps when loading the weights https://github.com/pytorch/vision/blob/master/torchvision/models/densenet.py#L203. I think we can perhaps just remove the call and make a comment notifying the user of clean up if necessary.

Reply:
In that case I agree with just removing this cleanup, the worst case scenario of having to clean up some extra models is better than accidentally deleting something important.